pkg/cloud/aws/actuators/machine/actuator.go

```
@@ -107,23 +112,32 @@ func (a *Actuator) isNodeJoin(scope *actuators.MachineScope, controlPlaneMachine
 			})
 
 			if err != nil {
-				return false, errors.Wrapf(err, "failed to create machine scope for machine %q", cm.Name)
+				a.log.V(2).Info("failed to create machine scope for machine", "machine-name", cm.Name, "machine-namespace", cm.Namespace)
+				continue
 			}
 
 			ec2svc := ec2.NewService(m.Scope)
 
-			ok, err := ec2svc.MachineExists(m)
+			controlplaneExists, err = ec2svc.MachineExists(m)
 			if err != nil {
-				return false, errors.Wrapf(err, "failed to verify existence of machine %q", m.Name())
+				a.log.V(2).Info("Failed to verify existence of control plane machine", "machine-name", m.Machine.Name, "machine-namespace", m.Machine.Namespace)
```

Thinking that this should return the error instead of `continuing`